pkg/framework/aries/default_js_wasm.go

```diff
@@ -23,7 +25,7 @@ func storeProvider() (storage.Provider, error) {
 }
 
 func transientStoreProvider() (storage.Provider, error) {
-	storeProv, err := jsindexeddb.NewProvider("temp")
+	storeProv, err := jsindexeddb.NewProvider("temp" + uuid.New().String())
```

Comment:
I don’t think we should create a new database every time.

Reply:
Yes - this will fill up storage needlessly.

Any reason why this can't be under the persistent store's namespace?

Reply:
Separate store, with a deterministic key under the same namespace.

Reply:
this is for transient db only, right now it is being used by multiple aries instances simultaneously, because of common db name `aries-temp`. btw, this is temp db should it be reused for new instances?

Reply:
reverted